src/components/Store.js

```diff
@@ -0,0 +1,30 @@
+// import { useState } from 'react'
+import initialStoreItems from '../data/StoreItems'
+
+const Store = () => {
+  // const [storeItems, setStoreItems] = useState(initialStoreItems)
+  // const [cartItems, setCartItems] = useState([])
+
+  return (
+    <header id="store">
+      <h1>Greengrocers</h1>
+      <ul className="item-list store--item-list">
+        {initialStoreItems.map(item => (
```

Comment:
This looks good and very much readable for me as I have done something very similar to this.